geotools/src/main/scala/geotrellis/geotools/GridCoverage2DConverters.scala

```diff
@@ -190,6 +196,15 @@ object GridCoverage2DConverters {
     new Envelope2D(null, xmin, ymin, (xmax - xmin), (ymax - ymin))
   }
 
+  def getGeotoolsCRS(crs: CRS): CoordinateReferenceSystem = {
+    crs.epsgCode match {
+      case Some(code) =>
+        authorityFactory.createCoordinateReferenceSystem(s"EPSG:${code}")
```

Comment:
The motivation is to make sure that [the coordinate order is well-specified](http://docs.geotools.org/latest/userguide/library/referencing/order.html).  The `authorityFactory` variable is not recreated, but since the variable `code` is unknown, this seems (to me) to be required.